multisrc/overrides/nepnep/mangasee/src/MangaSee.kt

```diff
@@ -15,4 +15,19 @@ class MangaSee : NepNep("MangaSee", "https://mangasee123.com", "en") {
         .readTimeout(1, TimeUnit.MINUTES)
         .writeTimeout(1, TimeUnit.MINUTES)
         .build()
+
+    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
+        if (query.startsWith("id:")) {
+            val id = query.substringAfter("id:")
+        } return client.newCall(GET("$baseUrl/manga/$id/"))
+                .asObservableSuccess() 
+                .map { 
+                response ->
+                    val manga = mangaDetailsParse(response) 
+                    manga.url = "/manga/$id/" 
+                    MangasPage(listOf(manga), false)
+                }
+        }
+        return super.fetchSearchManga(page, query, filters)
+    }
```

Comment:
Probably added a `}` as a mistake, so I fixed it. Check if it still works as expected.

```suggestion
override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
        if (query.startsWith("id:")) {
            val id = query.substringAfter("id:")
            return client.newCall(GET("$baseUrl/manga/$id/"))
                .asObservableSuccess() 
                .map { response ->
                    val manga = mangaDetailsParse(response) 
                    manga.url = "/manga/$id/" 
                    MangasPage(listOf(manga), false)
                }
        }
        
        return super.fetchSearchManga(page, query, filters)
    }
```